frontends/web/src/components/bitboxbase/connectbase.tsx

```diff
@@ -24,8 +24,14 @@ export interface ConnectedBaseProps {
     bitboxBaseID: string;
 }
 
+interface BlockInfoType {
+    blocks: number;
+    difficulty: number;
+    lightningAlias: string;
+}
+
 interface State {
-    blockInfo: string;
+    blockInfo: BlockInfoType;
```

Comment:
could also be `blockInfo?: BlockInfoType`, which is an alis for `BlockInfoType | undefined`, and then start with `undefined`. 

Then you know if you have a blockInfo or not do the workaround below with the dummy initial value.